index.js

```diff
@@ -0,0 +1,69 @@
+// global variables
+let searchCityForm = document.getElementById('searchCityForm');
+let errorMessage = document.getElementById('errorMessage');
+let city = document.getElementById('city');
+let temp = document.getElementById('temp');
```

Comment:
Is the `temp` DOM element a Text box? Label? Form? Number? String? When naming, be as specific as you can be